bin/inspec

```diff
@@ -115,11 +115,13 @@ class Inspec::InspecCLI < Inspec::BaseCLI # rubocop:disable Metrics/ClassLength
   def detect
     diagnose
 
-    runner = Inspec::Runner.new(opts)
     rel = File.join(File.dirname(__FILE__), *%w{.. lib utils detect.rb})
     detect_util = File.expand_path(rel)
-    runner.add_tests([detect_util])
-    runner.run
+    # exits on execution:
+    runner = Inspec::Runner.new(opts)
+    profile = Inspec::Profile.for_target(detect_util, opts)
+    runner.add_profile(profile)
```

Comment:
Good point, also a curious one. The advantage of passing multiple things to runner is that it can add all things to a run and then execute them. If we have that execution, would we need Profile.run anymore? I guess it's still great for convenience...